docs/public-networks/reference/cli/options.md

```diff
@@ -665,6 +665,37 @@ The default is `8551`.
     ```
 
 Reporting URL of an [Ethstats](../../../private-networks/how-to/deploy/ethstats.md) server.
+If specified without a port, the default port is 443 for SSL connections and 80 for non-SSL connections.
+
+### `ethstats-cacert-file`
+
+=== "Syntax"
+
+    ```bash
+    --ethstats-cacert-file=<FILE>
+    ```
+
+=== "Example"
+
+    ```bash
+    --ethstats-cacert-file=./root.cert
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    BESU_ETHSTATS_CACERT_FILE=./root.cert
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    ethstats-cacert-file="./root.cert"
+    ```
+
+Path to the root certificate authority (CA) certificate file of the Ethstats server specified by
+[`--ethstats`](#ethstats).
+This option is useful in non-production environments.
```

Comment:
Not sure I quite understand why it's useful in non-prod environments though.

Reply:
In production environment, a proper issues certificate is most likely to be used. This option is usually used when either self signed certificate is used or a custom CA is used to sign the server certificate.